Jenkinsfile

```diff
@@ -828,6 +828,7 @@ pipeline {
                     }
                     post {
                         always {
+                            junit testResults: 'bundles/junit-report.xml', allowEmptyResults: true
```

Comment:
I can disable the `allowEmptyResults` for now, so that it's clear if it failed

Perhaps we should do a `dir` (e.g.) too see what files are present, and if the location in which it's stored is correct